contributors/devel/sig-testing/testing.md

```diff
@@ -197,3 +197,61 @@ Please refer to [Integration Testing in Kubernetes](integration-tests.md).
 ## End-to-End tests
 
 Please refer to [End-to-End Testing in Kubernetes](e2e-tests.md).
+
+## Running your contribution in the Kubernetes CI
+Once you open a PR, [`prow`][prow-doc] runs pre-submit tests in CI.
+
+If you are not a [Kubernetes org member][member], another org member will need to run [`/ok-to-test`][ok-to-test] on your PR.
+
+Find out more about [other commands][prow-cmds] you can use to interact with prow through GitHub comments.
+
+### Troubleshooting a failure
+Click on `Details` and look at the [`gubernator`](gubernator.k8s.io/) output for the test.
+
+If the failure seems unrelated to the change you're submitting:
+- Is it a flake?
+  - Check if an issue has already been opened for that flake
+    - If not, open a new one (like [this example][new-issue-example]) and [label it `kind/flake`][kind/flake]
+  - Run [`/retest`][retest] on your PR to re-trigger the tests
```

Comment:
If it is a flake, and there is an issue for it, is there anything else I can do to aid with troubleshooting beyond running `/retest` on my PR?

eg: I personally use N=1 to say fine it's a flake, N=2 to spam retest, N=3 to say OK what can I do to help identify this flake and make it go away.  Sometimes that's looking at the actual failure and seeing if it's the same failure each time it happens.  Maybe it's looking back to see when this flake started to happen, and possibly identify the window of commits that should be explored further.  etc.

Reply:
Thanks, good point! I added a small section with pointers to what contributors can do - similar to working through artifacts I was cautious of being too prescriptive with a course of action that changes a lot depending on the context of the specific flake occurrence. Happy to add more though if you feel this is the right place to put more information.